resources/frontend_client/index_template.html

```diff
@@ -16,23 +16,34 @@
     <meta name="msapplication-config" content="/app/assets/img/browserconfig.xml">
     <meta name="theme-color" content="#ffffff">
     <meta name="apple-mobile-web-app-status-bar-style" content="default" />
+    <meta name="base-href" content="{{{baseHref}}}" />
+    <meta name="uri" content="{{{uri}}}" />
 
-    {{{embed_code}}}
+    {{{embedCode}}}
 
     <title>Metabase</title>
 
-    <base href={{{base_href}}} />
+    <base href="{{{baseHref}}}" />
 
+    <script type="application/json" id="_metabaseBootstrap">
+      {{{bootstrapJSON}}}
+    </script>
+
+    <script type="application/json" id="_metabaseLocalization">
+      {{{localizationJSON}}}
```

Comment:
these have been moved into separate inline JSON `<script>` tags so the JS `<script>` below can remain static and we can whitelist the hash